pkg/api/controllers/transaction_controller.go

```diff
@@ -40,13 +42,35 @@ func (ctl *TransactionController) CountTransactions(c *gin.Context) {
 func (ctl *TransactionController) GetTransactions(c *gin.Context) {
 	l, _ := c.Get("ledger")
 
+	var startTime, endTime time.Time
+	var err error
+	if c.Query(querystrings.KeyStartTime) != "" {
+		startTime, err = time.Parse(time.RFC3339, c.Query(querystrings.KeyStartTime))
+		if err != nil {
+			ResponseError(c, fmt.Errorf("%s %q: %w",
+				querystrings.ErrInvalidQueryValue, querystrings.KeyStartTime, err))
+			return
+		}
+	}
+
+	if c.Query(querystrings.KeyEndTime) != "" {
+		endTime, err = time.Parse(time.RFC3339, c.Query(querystrings.KeyEndTime))
+		if err != nil {
+			ResponseError(c, fmt.Errorf("%s %q: %w",
+				querystrings.ErrInvalidQueryValue, querystrings.KeyEndTime, err))
+			return
+		}
+	}
+
 	cursor, err := l.(*ledger.Ledger).GetTransactions(
 		c.Request.Context(),
 		query.After(c.Query("after")),
 		query.Reference(c.Query("reference")),
 		query.Account(c.Query("account")),
 		query.Source(c.Query("source")),
 		query.Destination(c.Query("destination")),
+		query.StartTime(startTime.UTC()),
```

Comment:
Why convert to UTC?

Reply:
You're right, there is no need to convert to UTC here. I need to convert them in the `transactionsQuery` function though, as RFC3339 timestamps on a different TZ than UTC doesn't work properly.

Reply:
Ok, strange.
Probably because the "timestamp" column is a varchar on database.
We should change this in a later release.

Reply:
Yes ! It might help :)